docs/plugins/index.rst

```diff
@@ -155,9 +155,6 @@ Autotagger Extensions
 Metadata
 --------
 
-:doc:`absubmit <absubmit>`
-   Analyse audio with the `streaming_extractor_music`_ program and submit the metadata to the AcousticBrainz server
-
```

Comment:
Also this should be brought back I suppose. Since a dockerized version would require to have exactly the same system of using `absubmit` to analyze and push to server/container and `acousticbrainz` then to retrieve it back from there.